apps/financial-aid/web-osk/src/routes/application/RelationshipstatusForm/RelationshipstatusForm.tsx

```diff
@@ -0,0 +1,157 @@
+import React, { useState, useContext } from 'react'
+import { Text } from '@island.is/island-ui/core'
+
+import {
+  ContentContainer,
+  Footer,
+  RadioButtonContainer,
+  SpouseInfo,
+} from '@island.is/financial-aid-web/osk/src/components'
+
+import { useRouter } from 'next/router'
+
+import useFormNavigation from '@island.is/financial-aid-web/osk/src/utils/useFormNavigation'
+import cn from 'classnames'
+import * as styles from './relationshipstatusForm.treat'
+
+import {
+  FamilyStatus,
+  isEmailValid,
+  NavigationProps,
+} from '@island.is/financial-aid/shared/lib'
+import { UserContext } from '@island.is/financial-aid-web/osk/src/components/UserProvider/UserProvider'
+
+const RelationshipstatusForm = () => {
+  const router = useRouter()
+
+  const { user, setUser } = useContext(UserContext)
+
+  const navigation: NavigationProps = useFormNavigation(
+    router.pathname,
+  ) as NavigationProps
+
+  const [hasError, setHasError] = useState(false)
+
+  const [acceptData, setAcceptData] = useState(false)
+
+  const options = [
+    {
+      label: 'Nei, ég er ekki í sambúð',
+      value: FamilyStatus.UNKNOWN,
+    },
+    {
+      label: 'Já, ég er í óstaðfestri sambúð',
+      value: FamilyStatus.UNREGISTEREDCOBAHITATION,
```

Comment:
This is probably breaking other naming convention but could we instead `FamilyStatus.UNREGISTERED_COBAHITATION` and same with other values in this enum?